pkg/ingester/ingester.go

```diff
@@ -306,6 +312,11 @@ func New(cfg Config, limits *validation.Overrides, activeGroupsCleanupService *u
 			Help: "The current number of series in memory.",
 		}, i.getMemorySeriesMetric)
 
+		promauto.With(registerer).NewGaugeFunc(prometheus.GaugeOpts{
+			Name: "cortex_ingester_ephemeral_series",
```

Comment:
[question] Have you considered adding a `memory_` to clarify they're in-memory? Similar to `cortex_ingester_memory_series `.

```suggestion
			Name: "cortex_ingester_memory_ephemeral_series",
```